www/src/pages/en/usage/trpc.md

```diff
@@ -1,7 +1,121 @@
+    </p>
+  </div>
+  <cite className="flex items-center justify-end">
+    <img
+      alt="Avatar of @t3dotgg"
+      className="w-12 mr-4 rounded-full bg-neutral-500"
+      src="https://avatars.githubusercontent.com/u/459267?v=4"
+    />
+    <div className="flex flex-col items-start">
+      <span className="mb-1 text-sm italic font-bold">Alex - creator of tRPC</span>
+      <a
+        href="https://twitter.com/alexdotjs"
+        target="_blank"
+        rel="noopener noreferrer"
+        className="text-sm"
+      >
+        @alexdotjs
+      </a>
+    </div>
+  </cite>
+</blockquote>
+
+## How do I use tRPC?
+
+With tRPC, you write Typescript functions on your backend, and then call them on your frontend. A simple tRPC procedure could look like this:
+
+```ts
+// server/routers/user.ts
+const userRouter = t.router({
+  getById: t.procedure.input(z.string()).query(({ ctx, input }) => {
+    return ctx.prisma.user.findFirst({
+      where: {
+        id: input,
+      },
+    });
+  }),
+});
+```
+
+What does this snippet do you might ask? Well, it's a tRPC procedure that takes a string as input that gets validated using Zod (which is the same validation library that we used for our [environment variables](./env-variables)), calls our database using [Prisma](./prisma) and returns the user whose `id` matches the one we passed in. You define your procedures in `routers` which is a collection of procedures which by convention should be related. You may have one router for `users`, one for `posts` and another one for `messages`. These routers can then be merged into a single, centralized `appRouter`:
+
+```ts
+// server/routers/index.ts
+const appRouter = t.router({
+  users: userRouter,
+  posts: postRouter,
+  messages: messageRouter,
+});
+
+export type AppRouter = typeof appRouter;
+```
+
+Notice that we only need to export our router's type definitions which means we are never importing any server code on our client.
+
+Now let's call the procedure on our frontend. tRPC provides a wrapper for `@tanstack/react-query` which let's you utilize the full power of the hooks they provide, but with the added benefit of having your API calls typed and inferred. We can call our procedures on our frontend like this:
+
+```tsx
+// pages/users/[id].tsx
+const UserPage = () => {
+  const userQuery = trpc.user.getById.useQuery("abc123");
+
+  return (
+    <div>
+      <h1>{user.data?.name}</h1>
+    </div>
+  );
+};
+```
+
+You'll immediately notice how good the autocompletion and typesafety is. Immediately when you have written `trpc.` your routers will show up, and eventually so will your procedures. You'll also get some red squiggly lines if your input doesn't match your validator that you defined on the backend.
```

Comment:
Instead of red squiggly lines we should say typescript errors as red squiggly lines aren't in all editors